### PR TITLE
Changed wiki link to new docs url

### PR DESCRIPTION
**Pull request by neauoire**

### CODE REVIEW

**Review by masinter — APPROVED** [review]
Thanks

**Comment by stumbo** [comment]
@masinter I realize I've been up to my eyeballs in other commitments the last couple months, but did I miss something?  Shouldn't the correct url be:  https://interlisp.org/doc/info/Using.html ?

**Comment by masinter** [comment]
@stumbo awaiting your Go Ahead.

File: README.md [hunk 1]
```diff
@@ -105,8 +105,7 @@ dump of your system located in your home directory named
 specify a specific image to run, Medley restores that image so that
 you can continue right where you left off.
 
-* [Using Medley Interlisp](https://github.com/Interlisp/medley/wiki/Using-Medley-Interlisp)
-
+* [Using Medley Interlisp](https://docs.interlisp.org/home/documentation/using)
```
Comment (stumbo) [inline README.md:108]: This leads to an old gitbook site.  We tried using gitbook but found it didn't serve our needs.  We're pushing all the user facing documentation on to our website, `www.interlisp.org`.  The correct URL would be `https://interlisp.org/doc/info/Using.html`

If you look at the following git repo" `https://github.com/Interlisp/Interlisp.github.io` you'll find all the current content on our website.  Please feel free to suggest updates to this material.

**Review by stumbo — CHANGES_REQUESTED** [review]

**Comment by neauoire** [comment]
Modified to point to the new, new url ;)

File: README.md (hunk 1 above)
Comment (stumbo) [reply README.md]: Thank you!

**Review by stumbo — APPROVED** [review]

**Pull request merged**